src/components/header/Header.component.tsx

```diff
@@ -1,2 +1,59 @@
+import bascet from '../../assets/basket-icon.svg';
+import line from '../../assets/line-icon.svg';
+import login from '../../assets/login-icon.svg';
+import logo from '../../assets/logo-icon.svg';
+import menu from '../../assets/menu-icon.svg';
+import moon from '../../assets/moon-icon.svg';
+import sun from '../../assets/sun-icon.svg';
+import userAdd from '../../assets/userAdd-icon.svg';
+
 import styles from './header.module.css';
-export const HeaderComponent = () => <header className={styles.header}>Header</header>;
+
+export function HeaderComponent() {
+    return (
+        <>
+            <header className={styles.header}>
```

Comment:
i think burger too early to use. 
Doing it on much smaller screens.

![Screenshot 2024-04-28 at 22 19 16](https://github.com/IgorFromUA/fe-react-2024/assets/115080771/d8ad1a82-d54b-4b47-8ae9-707eaa40ee89)

Reply:
fixed